packages/grafana-ui/src/components/TagsInput/TagsInput.mdx

```diff
@@ -0,0 +1,8 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
```

Comment:
Adding `mdx` files on their own won't enable docs, they also need to be added to the story file. Like so: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Button/Button.story.tsx#L13-L17 (also note the import at the top).

Reply:
ohhh that's good to know, probably Tobias mentioned that half a year ago and I forgot it again :D Thanks!

Reply:
Yeah, I was wondering earlier why do we have a few `mdx` files not connected to stories :D